app/src/main/java/com/hover/stax/settings/SettingsFragment.kt

```diff
@@ -113,6 +114,13 @@ class SettingsFragment : Fragment(), NavigationInterface {
         }
     }
 
+    private fun setupLearnCard() {
+        with(binding.staxLearn) {
+            learnFinances.setOnClickListener {  findNavController().navigate(R.id.action_navigation_home_to_wellnessFragment) }
```

Comment:
The navigation action here is incorrect. The source fragment here is the `settings`.